libs/blocks/section-metadata/section-metadata.css

```diff
@@ -266,3 +266,38 @@ main > .section[class*='-up'] > .content {
     padding-right: var(--grid-margins-width-10);
   }
 }
+
+.section.masonry-layout {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: var(--spacing-s);
+  padding-left: var(--grid-margins-width);
+  padding-right: var(--grid-margins-width);
+}
+
+.section.masonry-layout > div:not([class]),
+.section.masonry-layout > div:not([class]) > div.fragment,
+.section.masonry-layout > div:not([class]) > div.fragment > div.section {
+  display: grid;
+}
+
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+  .section.masonry-layout {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .section.masonry-layout .full-grid:first-child {
+    grid-column: span 2;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .section.masonry-layout {
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  .section.masonry-layout > div { grid-column: span 4; }
+  .section.masonry-layout .full-grid {grid-column: span 12; }
+  .section.masonry-layout .two-third-grid { grid-column: span 8; }
+  .section.masonry-layout .half-grid { grid-column: span 6; }
+}
```

Comment:
nit: keep media queries together, can move these to line 221

Reply:
@JasonHowellSlavin I have made the suggested changes.